serenata_toolbox/datasets/remote.py

```diff
@@ -0,0 +1,34 @@
+import configparser
+import os
+
+import boto3
+
+from serenata_toolbox.datasets.contextmanager import status_message
+
+
+class RemoteDatasets:
+
+    def __init__(self):
```

Comment:
if someone forgets to create the `config.ini` file (like I did) this should exit more gracefully instead of raising the `NoSectionError` (see details)
<details>
In[5]: datasets = Datasets('/Users/temporal/src/dsbr/serenata-data/')
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
/Users/temporal/anaconda3/envs/toolbox/lib/python3.6/configparser.py in _unify_values(self, section, vars)
   1137         try:
-> 1138             sectiondict = self._sections[section]
   1139         except KeyError:
KeyError: 'Amazon'
During handling of the above exception, another exception occurred:
NoSectionError                            Traceback (most recent call last)
<ipython-input-5-69a5a5f9fc40> in <module>()
----> 1 datasets = Datasets('/Users/temporal/src/dsbr/serenata-data/')
/Users/temporal/src/dsbr/serenata/serenata-toolbox/serenata_toolbox/datasets/__init__.py in __init__(self, local_directory)
     42 
     43         self.local = LocalDatasets(local_directory)
---> 44         self.remote = RemoteDatasets()
     45         self.downloader = Downloader(
     46             local_directory,
/Users/temporal/src/dsbr/serenata/serenata-toolbox/serenata_toolbox/datasets/remote.py in __init__(self)
     13         self.settings.read('config.ini')
     14         self.credentials = {
---> 15             'aws_access_key_id': self.settings.get('Amazon', 'AccessKey'),
     16             'aws_secret_access_key': self.settings.get('Amazon', 'SecretKey'),
     17             'region_name': self.settings.get('Amazon', 'Region')
/Users/temporal/anaconda3/envs/toolbox/lib/python3.6/configparser.py in get(self, section, option, raw, vars, fallback)
    779         """
    780         try:
--> 781             d = self._unify_values(section, vars)
    782         except NoSectionError:
    783             if fallback is _UNSET:
/Users/temporal/anaconda3/envs/toolbox/lib/python3.6/configparser.py in _unify_values(self, section, vars)
   1139         except KeyError:
   1140             if section != self.default_section:
-> 1141                 raise NoSectionError(section)
   1142         # Update with the entry specific variables
   1143         vardict = {}
NoSectionError: No section: 'Amazon'
</details>